<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item>
      <widget class="QListWidget" name="taskListWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="addTaskButton">
       <property name="text">
        <string>Dodaj zadanie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editTaskButton">
       <property name="text">
        <string>Edytuj zadanie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeTaskButton">
       <property name="text">
        <string>Usu≈Ñ zadanie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doneTaskButton">
       <property name="text">
        <string>Odznacz jako wykonane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="freedayButton">
       <property name="text">
        <string>Dni wolne</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="Menusettings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>Opcje</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
